packages/clay-card/src/CardWithHorizontal.tsx

```diff
@@ -86,6 +86,7 @@ export const ClayCardWithHorizontal: React.FunctionComponent<IProps> = ({
 							spritemap={spritemap}
 							trigger={
 								<button
+									aria-label="Open Dropdown"
```

Comment:
Unfortunately we can't add these to the `src` since we need the ability to localize the labels. Check out the [colorpicker](https://github.com/liferay/clay/blob/master/packages/clay-color-picker/src/index.tsx#L61-L64) and see how we add configureable aria-labels. Any sort of english text added to our components src needs to have the ability to be configured via props.